samples/python2/plane_tracker.py

```diff
@@ -61,7 +61,7 @@
 
 class PlaneTracker:
     def __init__(self):
-        self.detector = cv2.ORB( nfeatures = 1000 )
+        self.detector = cv2.ORB_create( nfeatures = 1000 )
```

Comment:
In this example we also need to replace `ch = cv2.waitKey(1)` with `ch = cv2.waitKey(1) & 0xFF` to catch keypresses on Linux.